pkg/codec/modifier.go

```diff
@@ -0,0 +1,17 @@
+package codec
+
+import (
+	"reflect"
+)
+
+type Modifier interface {
```

Comment:
Worth a comment?
FWIW you can run `make godoc` to run a local godoc server and see the generated HTML:
```
$ make godoc 
go install golang.org/x/tools/cmd/godoc@latest
# http://localhost:6060/pkg/github.com/smartcontractkit/chainlink-common/
godoc -http=:6060
```

Reply:
locally, I can't get some links to work, but I put what I thought I needed in the comments.  lmk if it's good enough or if I need to do something else.